README.md

```diff
@@ -150,6 +150,11 @@ Default to system default keychain.
 Regex or function that signals ignoring a file before signing.
 Default to `undefined`.
 
+`no-getekeeper-check` - *Boolean*
```

Comment:
I would recommend naming it `gatekeeper-assess` where `false` disables Gatekeeper check.